### PR TITLE
WIP: Setup a replication branch

### PR DESCRIPTION
**Pull request by xing-yang**
Make some basic changes for replication so DRBD driver development
can get started.

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: pkg/dock/proto/dock.proto [hunk 1]
```diff
@@ -58,6 +58,9 @@ service Dock {
     
     // Delete a volume attachment
     rpc DeleteAttachment (DeleteAttachmentOpts) returns (GenericResponse){}
+
+    // Create a replication
+    rpc CreateReplication(CreateReplicationOpts) returns (GenericResponse) {}
```
Comment (leonwanghui) [inline pkg/dock/proto/dock.proto]: You missed a blank letter here, is that the reason why the protobuf failed to generate the DockClient Interface?

Comment (xing-yang) [reply pkg/dock/proto/dock.proto]: Added it back and tried again. Still not generating.

Comment (xing-yang) [reply pkg/dock/proto/dock.proto]: What protobuf version are you using?

Comment (leonwanghui) [reply pkg/dock/proto/dock.proto]: I used to use v2.3 I guess, but I have upgraded it to 2.7

Comment (xing-yang) [reply pkg/dock/proto/dock.proto]: I think 2.7 is too old.  I had 2.x before but it says it need 3.x.

**Comment by xing-yang** [comment]
I'm going to merge this.  @xxwjj will continue with the work.

**Pull request merged**